documentation/release-notes/topics/version_3.rst

```diff
@@ -1,8 +1,57 @@
-.. include:: /release-notes/media.rst
-
 #########
 Version 3
 #########
+*******************************
+Hyperview 3.12 (April 14, 2023)
+*******************************
+This section covers significant changes and bug fixes in Hyperview 3.12.x since version 3.11.x.
+
+.. note:: This release has changes and improvements to the Data Collector software. It is strongly recommended to update your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+
+=============================================
+New Licensed Feature: Connectivity Management
+=============================================
+This is the first public phase of Connectivity. Hyperview Connectivity is designed to simplify network documentation for network engineers and IT professionals. Connectivity is a comprehensive tool that streamlines structured cabling management, network mapping, reporting, change tracking, and document asset association to network connections.
+
+Network documentation can be a daunting, time-consuming task that often involves searching for critical information and troubleshooting issues. Connectivity is here to alleviate these challenges by consolidating your network documentation into a centralized, easy-to-manage platform. With this release, you will be able to:
+
+1. Document connection information between assets
+2. Capture metadata, such as connection type, port configuration, length and any number of custom user-defined properties
+3. Define physical ports on all network-connected devices and associate those with connections
+4. Centrally manage associations in a Connectivity centric Layout tab available on network devices and patch panels
+5. Export, import and update all aspects of connection information individually or in bulk
+6. Set access policies on connections
+
+.. note:: Please contact our Sales team if you are interested in getting a Connectivity license.
```

Comment:
might be good to call it Connectivity Management here as well.

Reply:
Agreed. Fixed.